src/elements/content-sidebar/Sidebar.js

```diff
@@ -252,6 +253,24 @@ class Sidebar extends React.Component<Props, State> {
         }
     }
 
+    getDefaultPanel(): string | typeof undefined {
+        const { features } = this.props;
+
+        if (!isFeatureEnabled(features, 'panelSelectionPreservation')) {
+            return undefined;
+        }
+
+        return this.store.getItem(SIDEBAR_SELECTED_PANEL_KEY) || undefined;
```

Comment:
Suggested solution actually returns the opposite, so it will return the value from `LocalStore` when feature is disabled. I was thinking of minimizing the approach before, however even with your suggestion after inversion:
```
return isFeatureEnabled(features, 'panelSelectionPreservation')
        ? this.store.getItem(SIDEBAR_SELECTED_PANEL_KEY)
        : undefined;
```
we can get `null` if the key is not yet stored in local storage and I wanted this function to either return the value or `undefined`, so that I can directly pass it to `<SidebarPanels>`. That's why I went with a solution using guard clause.